pkg/api/common/versions_test.go

```diff
@@ -668,53 +671,63 @@ func Test_IsSupportedKubernetesVersion(t *testing.T) {
 }
 
 func Test_IsValidMinVersion(t *testing.T) {
-	orchestratorRelease := "1.16"
+	defaultVersion := RationalizeReleaseAndVersion(Kubernetes, "", "", false, false, false)
```

Comment:
these new foo allow us to UT the `IsValidMinVersion` helper func in a resilient way doesn't depend upon a particular version being supported or unsupported